packages/hint/src/lib/cli/analyze.ts

```diff
@@ -77,6 +78,57 @@ or set the flag --tracking=on|off`;
     printFrame(message);
 };
 
+const getHintsForTelemetry = (hints?: HintsConfigObject | (string | any)[]) => {
+    if (!hints) {
+        return null;
+    }
+
+    const normalizedHints = normalizeHints(hints);
+    const result = {} as HintsConfigObject;
+
+    for (const [hintId, severity] of Object.entries(normalizedHints)) {
+        result[hintId] = typeof severity === 'string' ? severity : (severity as [HintSeverity, any])[0];
+    }
+
+    return result;
+};
+
+const getHintsFromConfiguration = (configName: string, parentConfigs: string[] = []) => {
+    try {
+        const configuration = loadConfiguration(configName, parentConfigs);
+
+        return { ...getHintsFromConfigurations(configuration.extends, [configName, ...parentConfigs]), ...getHintsForTelemetry(configuration.hints) }; // eslint-disable-line no-use-before-define,@typescript-eslint/no-use-before-define
+    } catch (e) { // If the configuration doesn't exists, ignore it and returns an empty object.
+        return {};
+    }
+};
+
+const getHintsFromConfigurations = (configurations?: string[], parentConfigs: string[] = [], index: number = 0): any => {
```

Comment:
```suggestion
const getHintsFromConfigurations = (configurations?: string[], parentConfigs: string[] = [], index = 0): any => {
```

Reply:
In fact, the index is a leftover from debugging.

Reply:
Clearly I didn't follow it far enough then 😂